src/main/kotlin/no/nav/syfo/personstatus/api/v2/PersonOversiktStatusDTO.kt

```diff
@@ -19,6 +19,7 @@ data class PersonOversiktStatusDTO(
     val aktivitetskravSistVurdert: LocalDateTime?,
     val aktivitetskravActive: Boolean,
     val aktivitetskravVurderingFrist: LocalDate?,
+    val behandlerdialogUbehandlet: Boolean,
```

Comment:
Kan også utvide API-testene til å sjekke at dette flagget er satt riktig (men da må vi vel også utvide enhetens-oversikt-spørringen til å ta med disse)

Reply:
On it!